### ISSUE TITLE
Unregistering a dynamic capability has a typo in the method name

### ISSUE THREAD

**Issue by glennsarti** [microsoft/vscode-languageserver-node#521]
The unregistration method has a typo in the method name for the interface

https://github.com/microsoft/vscode-languageserver-node/blob/f0a625ec82/protocol/src/protocol.ts#L168-L170

`unregisterations` is not the real word, it should really be `unregistrations`

I don't expect this can be fixed given it's a breaking change, but for visibility... This is a typo.

Affect:

When I'm writing unit tests for my Language Server, I naturally use the correct spelling, however this causes unit test failures.

**Comment by dbaeumer** [comment]
@glennsarti you are correct that this would be breaking, especially this typo made it into the spec as well (see https://microsoft.github.io/language-server-protocol/specification#client_unregisterCapability)

So I will annotate the spec that at least this is obvious and we need to wait for a 4.x version to fix this.

**Comment by glennsarti** [comment]
Thanks @dbaeumer
